### PR TITLE
Add TFLite + Transformer example

### PR DESCRIPTION
**Pull request by calren**
Demonstrates applying an Effect generated by [TFLite's Artistic Style Transfer](https://www.tensorflow.org/lite/examples/style_transfer/overview) on a video using Transformer API.

https://github.com/user-attachments/assets/4c35b0de-d1e3-4e01-a461-6b355de51474

### CODE REVIEW

**Comment by calren** [comment]
cc @droid-girl @lethargicpanda

**Comment by droid-girl** [comment]
I checked-out the project. One minor comment: should it be possible to repeat the export with another style? At the moment the selector is available but the export button is disabled because the export completed already.
A proposal:
1) enable export button again when the first one completed 
2) if you want to keep it disabled, maybe also disable the style selector

File: samples/media/video/src/main/java/com/example/platform/media/video/StyleTransferShaderProgram.java
Comment (droid-girl) [inline samples/media/video/src/main/java/com/example/platform/media/video/StyleTransferShaderProgram.java:196]: remove commented out code?

File: samples/media/video/src/main/java/com/example/platform/media/video/StyleTransferShaderProgram.java
Comment (droid-girl) [inline samples/media/video/src/main/java/com/example/platform/media/video/StyleTransferShaderProgram.java:148]: Should we follow the same convention as in other classes and define a TAG const and use it instead of "Debug"?

File: samples/media/video/src/main/java/com/example/platform/media/video/StyleTransferShaderProgram.java
Comment (droid-girl) [inline samples/media/video/src/main/java/com/example/platform/media/video/StyleTransferShaderProgram.java:56]: Should we aim to convert it to Kotlin? Do you aim to keep this repo Kotlin only or any guidelines on this?

File: samples/media/video/src/main/java/com/example/platform/media/video/StyleTransferShaderProgram.java
Comment (calren) [reply samples/media/video/src/main/java/com/example/platform/media/video/StyleTransferShaderProgram.java:148]: that makes sense, will do

File: samples/media/video/src/main/java/com/example/platform/media/video/StyleTransferShaderProgram.java
Comment (calren) [reply samples/media/video/src/main/java/com/example/platform/media/video/StyleTransferShaderProgram.java:56]: yes, we should convert this to Kotlin even though we don't have any strict guidelines on keeping this repo kotlin only.

so i do this in a separate PR to keep it easier to review or just do it as part of this PR?

Comment (droid-girl) [reply samples/media/video/src/main/java/com/example/platform/media/video/StyleTransferShaderProgram.java:56]: I am ok with having a follow up PR with a Kotlin version as long as the repository owner is fine with it.

**Comment by calren** [comment]
addressed comments, please take another look!

>  should it be possible to repeat the export with another style?
I disabled the radio buttons when the export is started. I tried exporting with another style after an initial export, but the result looked glitchy, and I'm not sure why. It seems like only the first export gives a good result, and subsequent ones always looks glitchy.

**Review by droid-girl — APPROVED** [review]

**Comment by calren** [comment]
trying to make builds pass again on this repo before merging this (https://github.com/android/platform-samples/pull/185)

**Comment by calren** [comment]
@SigmanZero can you give this an LGTM? It still needs an approval from someone with write access on this repo

**Review by SigmanZero — APPROVED** [review]

**Comment by calren** [comment]
@ksemenova @lethargicpanda can you take a quick look at this PR as well?

checking out `gradle/libs.versions.toml` will probably be the most useful, to make sure we're using recommended dependencies

**Comment by calren** [comment]
merging this for now but @ksemenova @lethargicpanda let me know if there are concerns on the TFLite front and I can make follow up PRs to address

**Pull request merged**
